deploy-aks/azext_aks_deploy/dev/common/github_api_helper.py

```diff
@@ -22,10 +22,10 @@ def __init__(self, path, content):
         self.content = content
 
 
-def get_github_pat_token(display_warning=False):
+def get_github_pat_token(repo_name,display_warning=False):
```

Comment:
nit: The name "repo_name" feels too specific for this class, this could be any name that you want to give to the PAT token.